From 85de55c943cd9ec8a6290f6322ef5982f5157729 Mon Sep 17 00:00:00 2001
From: Lukas Bulwahn <lukas.bulwahn@gmail.com>
Date: Fri, 24 Mar 2023 11:08:48 +0100
Subject: [PATCH] mips: Remove obsolete configs IRQ_MSP_CIC and IRQ_MSP_SLP

Commit 1b00767fd8e1 ("MIPS: Remove PMC MSP71xx platform") removes all uses
of the config IRQ_MSP_CIC and IRQ_MSP_SLP.

Remove these two obsolete configs IRQ_MSP_CIC and IRQ_MSP_SLP.

Signed-off-by: Lukas Bulwahn <lukas.bulwahn@gmail.com>
Signed-off-by: Thomas Bogendoerfer <tsbogend@alpha.franken.de>
---
 arch/mips/Kconfig | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/arch/mips/Kconfig b/arch/mips/Kconfig
index fd7ed68cdb91..b91086b0ae22 100644
--- a/arch/mips/Kconfig
+++ b/arch/mips/Kconfig
@@ -1164,12 +1164,6 @@ config SYS_SUPPORTS_LITTLE_ENDIAN
 config MIPS_HUGE_TLB_SUPPORT
 	def_bool HUGETLB_PAGE || TRANSPARENT_HUGEPAGE
 
-config IRQ_MSP_SLP
-	bool
-
-config IRQ_MSP_CIC
-	bool
-
 config IRQ_TXX9
 	bool
 
-- 
2.39.1

